.circleci/config.yml

```diff
@@ -86,38 +102,48 @@ jobs:
               llvm-12-dev \
               ninja-build \
               pkg-config \
-              python3-setuptools \
-              sudo \
-              xsltproc
+              python3-setuptools
             pip3 install toml
           environment:
             DEBIAN_FRONTEND: noninteractive
-
+      - checkout
       - run:
           name: Build
           command: |
             cmake -G Ninja -B build/ -DWITH_TESTS=On
-            cmake --build build/ -j
+            cmake --build build/
+            # Testing rubbish:
+            cp test/ci.oid.toml build/testing.oid.toml
+      - persist_to_workspace:
+          root: .
+          paths:
+            - build/*
+            - extern/*
+            - types/*
 
+  test:
+    executor: big-boy
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: Install dependencies
+          command: |
+            sudo apt-get update
+            sudo apt-get install -y \
+              clang-12 \
+              libgflags-dev \
+              llvm-12-dev
       - run:
           name: Test
           environment:
             # disable drgn multithreading as tests are already run in parallel
             OMP_NUM_THREADS: 1
           command: |
             echo 0 | sudo tee /proc/sys/kernel/yama/ptrace_scope
-            cp test/ci.oid.toml build/testing.oid.toml
-            ctest --test-dir build/test/ --test-action Test -j$(nproc) \
```

Comment:
Did you perhaps try `$$(nproc)`? I haven't seen anything suggesting it would work but if the issue is it being substituted in two places it might. Hardcoding is definitely fine though.

Reply:
I've updated the PR's summary with an explanation about the number of cores to use. We can't actually use `nproc` because of some bugs with too high parallelism. 😢